translate/rus.json

```diff
@@ -364,7 +364,7 @@
 "subscribe":"подписаться",
 "target color":"цвет цели",
 "target opacity":"непрозрачность цели",
-"TB Hits":"попадания в ЭТ",
+"TB Hits":"Попадания в ЭТ",
```

Comment:
For some reason even though line 329 translation started with a capital letter in GUI and a small letter here, line 367 does not follow this rule at all for some reason. So I standardized them, so both of them start with a capital letter here and in the GUI.